plugins/python_3_script/help.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-[Python](https://www.python.org/) is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.6.5 and its standard library as well as the following 3rd party libraries: 
+[Python](https://www.python.org/) is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.8 and its standard library as well as the following 3rd party libraries: 
```

Comment:
my bad for the last comment this was meant to be python 3.8.1 not 3.8

from docker container 
```
python --version
Python 3.8.1
```